operations/http.go

```diff
@@ -27,12 +27,13 @@ import (
 // legacyClient manages requests to the API server endpoints, and unmarshaling the results into
 // usable structures.
 type legacyClient struct {
-	APIRoot    string
-	httpClient http.Client
-	User       string
-	APIKey     string
-	APIRootV2  string
-	UIRoot     string
+	APIRoot     string
+	httpClient  http.Client
+	User        string
+	APIKey      string
+	APIRootV2   string
+	UIRoot      string
+	environment string
```

Comment:
Is there a plan to use this for non-staging environments? If not we can rename to something like stagingEnvironment for clarity.

Reply:
changed to `stagingEnvironment`